build/lib/stylelint/vscode-known-variables.json

```diff
@@ -971,6 +971,13 @@
 		"--vscode-action-item-auto-timeout",
 		"--monaco-editor-warning-decoration",
 		"--animation-opacity",
-		"--chat-setup-dialog-glow-angle"
+		"--chat-setup-dialog-glow-angle",
```

Comment:
@lszomoru I think how this works is that these need to be in the manual 'others' section, not the auto-generated 'colors' section, so they got deleted when someone ran the update script.